src/forms/labels/labeled-field.js

```diff
@@ -20,6 +20,7 @@ import { hasInputError } from '../helpers'
  * @param {Boolean} [hideErrorLabel] - A boolean determining whether to hide the error label on input error (optional, default `false`)
  * @param {Function} [labelComponent=InputLabel] - A custom label component for the input
  * @param {Function} [errorComponent=InputError] - A custom error component for the input
+ * @param {Element Type} [as] - A string that determines the element type of the component (optional, default `div`)
```

Comment:
You can specify props as optional and their default automatically using JSDoc syntax. Refer to `labelComponent` above for how to accomplish that. While we're here, we might as well make that update to `hideErrorLabel` as well.